### DISCUSSION TITLE
Why on 3G network Sunshine/Moonlight is unusable, when Parsec is still playable?

### DISCUSSION THREAD

**Discussion by Degot** [LizardByte/community#492] [Sunshine]
I have remote location with 3G network only available. There is 60-90 ms latency between Host -> Client. Unfortunately, Sunshine/Moonlight are not usable with ~95% frame drops by network (according to debug output) , while I still can play games and use remote desktop with Parsec. What are significant differences between network protocols or Sunshine/Moonlight and Parsec?
How can I make Sunshine/Moonlight achieve the same performance as Parsec? Thank you.

**Comment by ReenigneArcher** [comment]
Parsec default settings are much lower quality. I don't know if you're using defaults though. They probably also have a larger buffer, which will result in more latency.

[a comment by an automated account is not shown]
